apps/auth/ssr/src/app/page.tsx

```diff
@@ -1,22 +1,48 @@
+import { cookies } from "next/headers";
 import { createCrossmint, CrossmintAuth } from "@crossmint/server-sdk";
 
-export default function Home() {
+export default async function Home() {
+    let isLoggedIn = false;
+    let userId: string | undefined;
+
     try {
         const crossmint = createCrossmint({
             apiKey: process.env.SERVER_CROSSMINT_API_KEY || "",
         });
         const crossmintAuth = CrossmintAuth.from(crossmint);
 
-        console.log(crossmint);
-        console.log(crossmintAuth);
+        const cookieStore = cookies();
+        const jwtCookie = cookieStore.get("crossmint-jwt")?.value;
+        const refreshCookie = cookieStore.get("crossmint-refresh-token")?.value;
+
+        if (refreshCookie != null) {
+            const { jwtToken, userId: fetchedUserId } = await crossmintAuth.getSession({
+                jwtToken: jwtCookie,
+                refreshToken: refreshCookie,
+            });
+            isLoggedIn = true;
+            userId = fetchedUserId;
+            console.log("jwt", jwtToken);
+            console.log("userId", userId);
+        } else {
+            console.log("user not logged in");
+        }
```

Comment:
this is probably superfluous. As this will show up in client logs maybe we remove
```suggestion
        }
```